### PR TITLE
Create Kitchen Request Model

### PR DESCRIPTION
**Pull request by thywang**
## Notion ticket link

[Create Kitchen Request Model](https://www.notion.so/uwblueprintexecs/Create-Kitchen-Request-Model-70c293e51d494f478883c96855738e3e?pvs=4)

## Implementation description
* Create Prisma schema for kitchen request model as per [Figma board](https://www.figma.com/file/99N86wIdlsQuQ2Jat9kv0k/ABTC-Architecture-Design?type=whiteboard&node-id=1-34&t=GE7nuttz58QlLy4P-0)

## Steps to test
1. Create endpoint to test (similar to `createUser` in `backend/typescript/services/implementations/userService.ts`)

## What should reviewers focus on?
* Field types are correct
* Relationship with users table is correct

## Checklist
- [x] My PR name is descriptive and in imperative tense
- [x] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [x] I have run the appropriate linter(s)
- [x] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

File: backend/typescript/prisma/schema.prisma [hunk 1]
```diff
@@ -12,25 +12,25 @@ model test {
...
 }
 
-//tina-emma/kitchen-request-model
 model kitchenRequest {
   id            String   @id @default(auto()) @map("_id") @db.ObjectId
   v             Int      @map("__v")
   requestName   String
-  requester     User     @relation(fields: [requesterId], references: [id])
-  requesterId   String     
+  requester     users    @relation(fields: [requesterId], references: [id])
```
Comment (thywang) [inline backend/typescript/prisma/schema.prisma:27]: Not sure if we'd like to keep this `requester` relation & `kitchenRequest` in users model above, or have only `requesterId` as the link to the users model for simplicity.

File: backend/typescript/prisma/schema.prisma [hunk 2]
```diff
@@ -12,11 +12,25 @@ model test {
...
- }
\ No newline at end of file
+  id             String           @id @default(auto()) @map("_id") @db.ObjectId
+  v              Int              @map("__v")
+  authId         String
+  email          String
+  firstName      String
+  lastName       String
+  role           String
+  kitchenRequest kitchenRequest[]
```
Comment (anmoltyagi1) [inline backend/typescript/prisma/schema.prisma]: Lets make the `kitchenRequest` plural, very small nitpick

File: backend/typescript/prisma/schema.prisma [hunk 3]
```diff
@@ -12,11 +12,25 @@ model test {
...
+}
+
+model kitchenRequest {
+  id            String   @id @default(auto()) @map("_id") @db.ObjectId
+  v             Int      @map("__v")
```
Comment (anmoltyagi1) [inline backend/typescript/prisma/schema.prisma]: Sorry to not clarify earlier, but we can ditch this `v` field

File: backend/typescript/prisma/schema.prisma (hunk 1 above)
Comment (anmoltyagi1) [reply backend/typescript/prisma/schema.prisma:27]: Looks good!

**Review by anmoltyagi1 — CHANGES_REQUESTED** [review]
Just the two changes, nothing big!

**Review by anmoltyagi1 — APPROVED** [review]
Looks good!!🚀

**Pull request merged**